packages/components/src/presentation-components/properties/NavigationPropertyTargetSelector.scss

```diff
@@ -9,4 +9,7 @@
     outline: none;
     padding: 0;
   }
+  > input::placeholder {
```

Comment:
Do we really need this? Can we stick to what browser defaults to?

Reply:
yes, otherwise there can be UI inconsistencies, for example: 
![image](https://github.com/iTwin/presentation/assets/124659623/17f49e7c-9913-4227-a882-992c1c9f640b)